src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2559,6 +2559,21 @@ public static void DCS_NonGenericDictionaryOfStringInt32()
 
     #endregion
 
+    [Fact]
+    public static void BasicRoundTripResolveDTOTypes()
+    {
+        ObjectContainer instance = new ObjectContainer(new DTOContainer());
+        XmlObjectSerializer serializer = new DataContractSerializer(typeof(ObjectContainer), null, null, null, int.MaxValue, false, false, new DTOResolver());
```

Comment:
Why this is declared as XmlObjectSerializer? How about just using 'var' instead?